src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/entities/BailCaseFieldDefinition.java

```diff
@@ -253,7 +253,10 @@ public enum BailCaseFieldDefinition {
         "detentionFacility", new TypeReference<String>(){}),
     APPLICATION_SUBMISSION_DOCUMENT(
         "applicationSubmissionDocument", new TypeReference<List<IdValue<DocumentWithMetadata>>>(){}),
-    ;
+    BAIL_SUMMARY(
+        "uploadBailSummaryDocs", new TypeReference<List<IdValue<DocumentWithDescription>>>(){}),
```

Comment:
Please follow the convention camelCase -> UPPERCASE_UNDERSCORE_SEPARATED.

Reply:
Isn't it already following the convention?
The enum constants are ```BAIL_SUMMARY``` and ```BAIL_SUMMARY_WITH_METADATA``` and their values are ```"uploadBailSummaryDocs"``` and ```"uploadBailSummaryMetadata"``` like all other enum constants

Reply:
uploadBailSummaryDocs -> UPLOAD_BAIL_SUMMARY_DOCS and uploadBailSummaryMetadata -> UPLOAD_BAIL_SUMMARY_METADATA